NashTeamAlpha/BangazonWeb
# Gr documenting

## Pull Request Description

1. Added documentation to any class that didn't have it previously, used our ticket record to determine who had created which class.
1. Removed unnecessary usings form Program.cs.
1. Replaced "Constructor in class" connotations with "Methods in Class" to keep all documentation concise and consistent.
1. Added some documentation to our Site.js event listeners to better describe what their purpose it. 

## Steps to Test

1. Check for spelling or syntax errors.
1. Check that documentation is properly accredited and connotated, if there is something inaccurate reference it in your comment.  

## Link to Feature Ticket

#91 
#92
